std/event/tcp.zig

```diff
@@ -123,10 +123,17 @@ pub async fn connect(loop: *Loop, _address: *const std.net.Address) !std.os.File
 }
 
 test "listen on a port, send bytes, receive bytes" {
-    if (builtin.os != builtin.Os.linux) {
-        // TODO build abstractions for other operating systems
-        return;
+    // TODO build abstractions for other operating systems
+    const skip_test: bool = switch (builtin.os) {
+        builtin.Os.linux => false,
+        //builtin.Os.macosx, builtin.Os.ios => false,
+        else => true,
+    };
+
+    if (skip_test == true) {
+        return error.skip;
```

Comment:
It may be better to use a name such as `error.SkipZigTest` (open to better suggestions) since this is global and we don't want a user to possibly use this name in their own code.

Reply:
@tiehuis Thanks!

I thought about that, but if used in user code, wouldn't it be caught before hand?

You bring up a good point and perhaps we could rename it to `error.skip_test` :-)